core/platform/taskqueue/dev_mode_taskqueue_services.py

```diff
@@ -60,8 +70,15 @@ def _task_handler(url, payload, queue_name, task_name=None):
 CLIENT = cloud_tasks_emulator.Emulator(task_handler=_task_handler)
 
 
+# In the type annotation below, payload is of type Dict[str, Any] because
+# the payload here has no constraints.
 def create_http_task(
-        queue_name, url, payload=None, scheduled_for=None, task_name=None):
+        queue_name: str,
+        url: str,
+        payload: Optional[Dict[str, Any]] = None,
```

Comment:
Why it can be optional?

Reply:
It mocks the function [here](https://github.com/oppia/oppia/blob/a4a3c2b7c673e928264919b595f51222ddd8e2c5/core/platform/taskqueue/cloud_taskqueue_services.py#L39) for dev mode. Therefore the types should be same.